src/test/java/org/fundacionjala/coding/nestor/CamelCaseTest.java

```diff
@@ -0,0 +1,74 @@
+package org.fundacionjala.coding.nestor;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test for {@link CamelCase}.
+ */
+public class CamelCaseTest {
+
+  /**
+   * Verify if two words can be converted to TwoWord.
+   */
+  @Test
+  public void testTwoWords() {
+    CamelCase solution = new CamelCase();
```

Comment:
You are repeating this line of code in all your tests.
Please move it to a @Before method.